test/spec-validation.js

```diff
@@ -0,0 +1,20 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/loopback-next-hello-world
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+'use strict';
+
+const HelloWorldApp = require('../lib/application').HelloWorldApp;
+const validateApiSpec = require('@loopback/testlab').validateApiSpec;
+
+describe('Api Spec', () => {
+  let app = new HelloWorldApp();
+  let apiSpec = app.getApiSpec();
```

Comment:
These two lines should go either to a `before` hook, or directly into the `it('is valid')` test case.